python/test/unit/io/test_adios2.py

```diff
@@ -270,6 +270,161 @@ def partitioner(comm, nparts, local_graph, num_ghost_nodes):
             dest = np.full(len(cells), comm.rank, dtype=np.int32)
             return adjacencylist(dest)
 
+<<<<<<< HEAD
```

Comment:
This line looks like a git conflict marker that shouldn't be here (needs reconciliation after merging from HEAD?)